libraries/c_sdk_v4/standard/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+afr_module(NAME "coreMQTT" INTERFACE)
```

Comment:
For OCW visibility, the module cannot be an `INTERFACE`

You can use my branch for MQTT integration test porting as reference :) 
https://github.com/aggarw13/amazon-freertos/blob/mqtt/integration-tests-port/libraries/c_sdk_v4/standard/CMakeLists.txt

Reply:
Thanks. That's a good suggestion.

Reply:
Changed the plan to update this PR itself. Addressed it